simple-cipher/simple-cipher.js

```diff
@@ -0,0 +1,43 @@
+function Cipher (key) {
+    this.alphabet = "abcdefghijklmnopqrstuvwxyz".split('');
+    if (key && /^[a-z]+$/.test(key)) {
+        this.key = key
+    } else if (!/^[a-z]+$/.test(key)){
+        throw new Error('Bad key');
+    } else {
+        this.key = Array(15).join("a").split('').map((letter) => {
```

Comment:
> If someone doesn't submit a key at all, generate a truly random key of at least 100 characters in length

From the exercise description.

special subtask from me: implement this via recursive function